templates/partials/summary/list-summary.html

```diff
@@ -1,38 +1,42 @@
 {% from "components/summary/_macro.njk" import onsSummary %}
 
-{% if list.list_items | length > 0 %}
-
-  {% if list.editable %}
-    {% set headers = ["Name of person", "Action"] %}
-    {% set rows = map_list_collector_config(
-      list.list_items,
-      "person",
-      _("Change"),
-      _("Change details for {item_name}"),
-      _("Remove"),
-      _("Remove {item_name}")
-    ) %}
-  {% else %}
-    {% set headers = ["Name of person"] %}
-    {% set rows = map_list_collector_config(
-      list.list_items,
-      "person"
-    ) %}
-  {% endif %}
+{% if list.editable %}
+  {% set headers = ["Name of person", "Action"] %}
+  {% set rows = map_list_collector_config(
+    list.list_items,
+    "person",
+    _("Change"),
+    _("Change details for {item_name}"),
+    _("Remove"),
+    _("Remove {item_name}")
+  ) %}
+{% else %}
+  {% set headers = ["Name of person"] %}
+  {% set rows = map_list_collector_config(
+    list.list_items,
+    "person"
+  ) %}
+{% endif %}
 
-  {{ onsSummary({
-    "withinQuestion": true,
-    "summaries": [
-      {
-        "groups": [
-          {
-            "headers": headers,
-            "rows": rows,
-          }
-        ]
-      }
-    ],
-    "classes": "u-mt-s u-mb-m"
-  }) }}
+{% set groups_config = {
+  "groupTitle": summary_title,
+  "headers": headers,
+  "rows": rows,
+  "summaryLink": {"url": add_link, "text": add_link_text, "dataQa": "add-item-link"},
+} %}
 
+{% if empty_list_text %}
+  {% do groups_config.update({"placeholderText": empty_list_text}) %}
 {% endif %}
```

Comment:
The empty list text and the title is not relevant to list collector pages, so either we update both of them to be an `if` statement to update the config if it exists or we just pass it to the DS with null values which is handled as expected.

So we could remove the if and just simplify to this where all config is always passed even when it may not exist and let the component handle it.
```jinja2
{% set groups_config = {
  "groupTitle": summary_title,
  "headers": headers,
  "rows": rows,
  "summaryLink": {"url": add_link, "text": add_link_text, "dataQa": "add-item-link"},
  "placeholderText": empty_list_text
} %}
```

Thoughts? @petechd  @ajmaddaford

Reply:
I will simplify it since Jon told me that we will never use `placeholderText` element in census.

Reply:
This is simpler, since placeholderText won't be created when there is no value I would go with this approach.